src/connectors/rating-menu/connectRatingMenu.js

```diff
@@ -129,31 +132,77 @@ export default function connectRatingMenu(renderFn, unmountFn = noop) {
   checkRendering(renderFn, withUsage());
 
   return (widgetParams = {}) => {
-    const { attribute, max = 5 } = widgetParams;
+    const { attribute, max = 5, step = 1 } = widgetParams;
     let sendEvent;
 
     if (!attribute) {
       throw new Error(withUsage('The `attribute` option is required.'));
     }
 
     const getRefinedStar = state => {
-      const refinements = state.getDisjunctiveRefinements(attribute);
+      const values = state.getNumericRefinements(attribute);
 
-      if (!refinements.length) {
+      if (!values['>=']?.length) {
         return undefined;
       }
 
-      return Math.min(...refinements.map(Number));
+      return values['>='][0];
+    };
+
+    const getFacetsMaxDecimalPlaces = facetResults => {
```

Comment:
What does it mean by `places` here? What about `getFacetsMaxDecimalLength`?

Reply:
"Decimal place" is the arithmetic term for the position of a digit to the right of the decimal point in a decimal number. (https://en.wiktionary.org/wiki/decimal_place). We can stick with the informal term "decimals" (`getFacetsMaxDecimals`) if you find it clearer.

Reply:
ah didn't know about that term. Let's go for the official one, then :)